site_root/input/resources/profiles/StructureDefinition-kfdrc-condition.json

```diff
@@ -0,0 +1,54 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition",
+    "version": "0.1.0",
+    "name": "kfdrc-condition",
+    "title": "KFDRC Condition",
+    "status": "draft",
+    "fhirVersion": "4.0.0",
+    "kind": "resource",
+    "abstract": false,
+    "type": "Condition",
+    "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Condition",
+    "derivation": "constraint",
+    "differential": {
+      "element": [
+        {
+          "id": "Condition",
+          "path": "Condition"
+        },
+        {
+          "id": "Condition.extension",
+          "path": "Condition.extension"
+        },
+        {
+          "id": "Condition.extension:age-at-event",
+          "path": "Condition.extension",
+          "sliceName": "age-at-event",
+          "sliceIsConstraining": false,
+          "label": "age_at_event",
+          "short": "Age time of event",
+          "definition": "Age (in days since birth) of the Participant at time of event",
```

Comment:
Can you update the `short` and `definition` to be a little more specific (something like time of condition observation/onset?)